internal/cmd/local/command_kafka_start.go

```diff
@@ -127,46 +127,109 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
+					HostPort: ports.KafkaRestPorts[0],
+				},
+			},
+			natPlaintextPorts[0]: []nat.PortBinding{
+				{
+					HostIP:   localhost,
+					HostPort: ports.PlaintextPorts[0],
+				},
 			},
 		},
-		natPlaintextPort: []nat.PortBinding{
-			{
-				HostIP:   localhost,
-				HostPort: ports.PlaintextPort,
+	}
+
+	config2 := &container.Config{
+		Image:    dockerImageName,
+		Hostname: "broker2",
+		Cmd:      containerStartCmd,
+		ExposedPorts: nat.PortSet{
+			natKafkaRestPorts[1]: struct{}{},
+			natPlaintextPorts[1]: struct{}{},
+		},
+		Env: getContainerEnvironmentWithPorts(ports, 1),
+	}
+	hostConfig2 := &container.HostConfig{
+		NetworkMode: container.NetworkMode("confluent-local-network"),
+		PortBindings: nat.PortMap{
+			natKafkaRestPorts[1]: []nat.PortBinding{
+				{
+					HostIP:   localhost,
+					HostPort: ports.KafkaRestPorts[1],
+				},
+			},
+			natPlaintextPorts[1]: []nat.PortBinding{
+				{
+					HostIP:   localhost,
+					HostPort: ports.PlaintextPorts[1],
+				},
 			},
 		},
-	},
 	}
 
-	createResp, err := dockerClient.ContainerCreate(context.Background(), config, hostConfig, nil, platform, confluentLocalContainerName)
+	createResp1, err := dockerClient.ContainerCreate(context.Background(), config1, hostConfig1, nil, platform, "broker1")
+	if err != nil {
+		return err
+	}
+	log.CliLogger.Tracef("Create confluent-local container for broker 1 success")
+	err = dockerClient.ContainerStart(context.Background(), createResp1.ID, types.ContainerStartOptions{})
 	if err != nil {
 		return err
 	}
-	log.CliLogger.Tracef("Create confluent-local container success")
 
-	err = dockerClient.ContainerStart(context.Background(), createResp.ID, types.ContainerStartOptions{})
+	multiBroker, err := cmd.Flags().GetBool("multi-broker")
 	if err != nil {
 		return err
 	}
+	if multiBroker {
+		createResp2, err := dockerClient.ContainerCreate(context.Background(), config2, hostConfig2, nil, platform, "broker2")
+		if err != nil {
+			return err
+		}
+		log.CliLogger.Tracef("Create confluent-local container for broker 2 success")
+		err = dockerClient.ContainerStart(context.Background(), createResp2.ID, types.ContainerStartOptions{})
+		if err != nil {
+			return err
+		}
 
-	output.Printf("Started Confluent Local container %v.\nTo continue your Confluent Local experience, run `confluent local kafka topic create test` and `confluent local kafka topic produce test`.\n", getShortenedContainerId(createResp.ID))
+		output.Printf("Started Confluent Local containers %v, %v.\nTo continue your Confluent Local experience, run `confluent local kafka topic create test` and `confluent local kafka topic produce test`.\n", getShortenedContainerId(createResp1.ID), getShortenedContainerId(createResp2.ID))
+	} else {
+		output.Printf("Started Confluent Local containers %v.\nTo continue your Confluent Local experience, run `confluent local kafka topic create test` and `confluent local kafka topic produce test`.\n", getShortenedContainerId(createResp1.ID))
```

Comment:
This PR didnt introduce this, but in testing this I think that  the second sentence should be the last thing output, and maybe add a couple of newlines, e.g.:
```
Started Confluent Local containers da0e273845, 934727acfa.

+------------------+---------------+
| Broker Ports     | [58583 58584] |
| Controller Ports | [58585 58586] |
| Kafka Rest Ports | [8082 9092]   |
| Plaintext Ports  | [58581 58582] |
+------------------+---------------+

To continue your Confluent Local experience, run `confluent local kafka topic create test` and `confluent local kafka topic produce test`.
```

Reply:
changed to only display krest port and plaintext ports as others are internal.
used comma-delimited format